apis/zalando.org/v1/types.go

```diff
@@ -81,21 +79,33 @@ const (
 type RouteGroupBackend struct {
 	// Name is the BackendName that can be referenced as RouteGroupBackendReference
 	Name string `json:"name"`
-	// Type is one of "service|shunt|loopback|dynamic|lb|network"
+	// Type of the backend.
+	// `service`- resolve Kubernetes service to the available Endpoints belonging to the Service, and generate load balanced routes using them.
```

Comment:
Unfortunately kubebuilder does not support markdown lists. I put allowed values in backticks and made each description a complete sentence (see PR description for kubectl explain output).